From f6c5366e11fa87bb66b9fdeffa4ea81269d54f1f Mon Sep 17 00:00:00 2001
From: dziie-id <alfindp@gmail.com>
Date: Sun, 1 Feb 2026 23:59:28 +0700
Subject: [PATCH 02/19] Create README.md

---
 README.md | 2 ++
 1 file changed, 2 insertions(+)
 create mode 100644 README.md

diff --git a/README.md b/README.md
new file mode 100644
index 0000000..7bb3c9d
--- /dev/null
+++ b/README.md
@@ -0,0 +1,2 @@
+- Enable Aux
+- Remove Thermal Throtle
--
2.53.0
